src/main/java/edu/kis/vh/nursery/HanoiRhymer.java

```diff
@@ -1,17 +1,18 @@
 package edu.kis.vh.nursery;
 
-public class HanoiRhymer extends defaultCountingOutRhymer {
+public class HanoiRhymer extends DefaultCountingOutRhymer {
 
-int totalRejected = 0;
+       public static final int INT2 = 0;
+    int totalRejected = INT2;
```

Comment:
should be private!
INT2 is a bad name.
0 is ZERO! If there is no other meaning constant is redundant